pages/linux/dconf-write.md

```diff
@@ -0,0 +1,24 @@
+# dconf write
+
+> Write a value to a dconf database path.
+> More information: <https://https://developer.gnome.org/dconf>.
+
+- Write a string to a dconf path (note the nested quotes):
+
+`dconf write {{/example/dconf/path}} "'{{Example Value}}'"`
+
+- Write a boolean to a dconf path:
+
+`dconf write {{/example/dconf/path}} {{true}}`
+
+- Write an integer to a dconf path:
+
+`dconf write {{/example/dconf/path}} {{16}}`
+
+- Write an array to a dconf path:
+
+`dconf write {{/example/dconf/path}} '[{{"My First Value", "My Second Value"}}]'`
+
+- Write an empty array to a dconf path:
+
+`dconf write {{/example/dconf/path}} "@as []"`
```

Comment:
```suggestion
`dconf write {{/example/dconf/path}} "{{@as []}}"`
```

Reply:
Since this example shows how to enter an empty array, this should be entered exactly as written

Reply:
Oh, I see.  Then never mind.

Reply:
not too sure about this one, @makusu2 what do you think

I think it doesn't need to be since its not a "variable" its writing a blank array

Reply:
Feel free to resolve